docs/shortcuts.md

```diff
@@ -0,0 +1,35 @@
+Shortcuts
+=========
+
+Like [Django's shortcuts][django-shortcuts], `django_toolkit.shortcuts` provide
+some helper functions to day-to-day development.
+
+[django-shortcuts]: https://docs.djangoproject.com/en/1.10/topics/http/shortcuts/
+
+
+### get_oauth2_app
+
+`django_toolkit.shortcuts.get_oauth2_app`
+
+Return the Application object from a Django Rest Framework request when
+authenticated with a Django OAuth Toolkit access token, or return None when it
```

Comment:
Good. What about italic or bold instead of backticks for the "Django OAuth Toolkit" name? I could do the same for "Django" and "Django Rest Framework".